### ISSUE TITLE
Add accessibility testing to JupyterHub testing tools

### ISSUE THREAD

**Issue by bl-aire** [jupyterhub/binderhub#1540]
### Add accessibility testing to JupyterHub testing tools 

The importance of building a product with accessibility in mind from the design/development phase cannot be overemphasized. There’s this option and then there is the option of building a product, deploying it and then looking for means to make it accessible.

The former saves time and resources that is why I am proposing this solution. 

### Current behavior 

Currently, no accessibility testing tool exists in JupyterHub's testing tools that can be used during the development phase. Accessibility testing tools are being used in the browser to improve how accessible the deployed site is.

### Proposed solution 

By using Selenium tool/library, accessibility testing tools like axe is integrated into the development testing so developers will be able take note if guidelines for accessibility aren’t being followed and make corrections before the product gets deployed.

Selenium is an open source automation testing tool with support for a couple of scripting languages like Python and JavaScript, etc. One can choose a script depending on the application to be tested. Selenium Python bindings provide a convenient API to access Selenium WebDrivers like Firefox, Ie, Chrome,etc.

### Alternative options

I am currently researching on alternative approaches and implementing this solution so I would love to have this issue assigned to me please. Thank you 

### Who would use this feature?

Implementing this feature will help developers and product owners(companies) save time and resources respectively.

**Comment by bl-aire** [comment]
@minrk 

I would love to have this issue assigned to me if it is approved please.

**Comment by sgibson91** [comment]
Hi @bl-aire did you mean to open this here? This is the BinderHub project repo, JupyterHub lives over here: https://github.com/jupyterhub/jupyterhub

**Comment by bl-aire** [comment]
> Hi @bl-aire did you mean to open this here? This is the BinderHub project repo, JupyterHub lives over here: https://github.com/jupyterhub/jupyterhub

Ohh sorry about that..I made a mistake. I can create the issue again in the right repository.

**Comment by sgibson91** [comment]
No problem, thank you! I will close this one 🙂

**Comment by bl-aire** [comment]
> No problem, thank you! I will close this one 🙂 

Thanks ☺️
